packages/platform-android/src/commands/runAndroid/tryLaunchAppOnDevice.ts

```diff
@@ -4,14 +4,14 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow
  */
 
 import {spawnSync} from 'child_process';
 import {logger, CLIError} from '@react-native-community/cli-tools';
 
 function tryLaunchAppOnDevice(
-  device?: string,
+  device?: string | void,
+  // @ts-ignore
```

Comment:
can you add a comment on what's wrong?

Reply:
Added a comment, what should be done to get rid of the suppression